poc/src/app/common/shared/components/chart-popups/chart-popups.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, Input } from '@angular/core'
+import { PopupsStoreService } from '@src/app/common/shared/services/popups-store.service'
+import { ChartWrapperPopupsService } from '@src/app/common/shared/components/chart-wrapper-base/chart-wrapper-popups.service'
+
+@Component({ selector: 'lw-chart-popups', templateUrl: './chart-popups.component.html', styleUrls: ['./chart-popups.component.less'] })
+export class ChartPopupsComponent {
+  @Input() chartWrapperPopupsService: ChartWrapperPopupsService
```

Comment:
Why do you pass service via input? Can you inject it?

Reply:
Fixed